css/style.css

```diff
@@ -254,10 +254,14 @@ button:hover {
 	font-size: 12pt;
 	font-weight: 700;
 	border: 0;
-	background-color: transparent;
+	background-color: white;
 	min-height: initial;
 }
 
+.stack form {
+	width: 100%;
+}
+
 .stack h2 button,
 .stack .stack-actions {
```

Comment:
without `widht: 100%` it looks like this:
![screenshot_20170519_185818](https://cloud.githubusercontent.com/assets/16517095/26258519/4676ea18-3cc5-11e7-9ce5-9f5ac6333ea0.png)